### PR TITLE
Tree boss ability assets

### PR DESCRIPTION
**Pull request by tigerdong**

### CODE REVIEW

File: characterEntities/Entity.java [hunk 1]
```diff
@@ -429,7 +429,7 @@ public void resetAbilities() {
 			Ability ability = abilities.get(i);
 			if (ability != null) {
 				ability.reset();
-				ability.resetCooldown();
+				ability.clearCooldown();
```
Comment (CathyWrHua) [inline characterEntities/Entity.java:432]: :P

Comment (tigerdong) [reply characterEntities/Entity.java:432]: :U

Comment (CathyWrHua) [reply characterEntities/Entity.java:432]: ty for fixing my bug

**Comment by CathyWrHua** [comment]
As discussed, might want to refactor before we build the boss anymore. But this pr lgtm

**Pull request merged**